conference.js

```diff
@@ -701,6 +704,14 @@ export default {
             return;
         }
 
+        if (!mute && shouldShowModeratedNotification(MEDIA_TYPE.AUDIO, state)) {
```

Comment:
This shouldn't be necessary, it's handled by `setAudioMuted`: https://github.com/jitsi/jitsi-meet/blob/1a34ed9a2d12cbda0d436f7a1843fe6b9fd79612/react/features/base/media/actions.ts#L62